### PR TITLE
Bugfix for creating ec2_metric_alarm without specifying dimensions:

### PR DESCRIPTION
**Pull request by fvant**
Adding {} to module.params.get does not work

### CODE REVIEW

**Comment by gregdek** [comment]
@zeekin Please review this PR to make sure it adheres to the following guidelines:

http://docs.ansible.com/developing_modules.html#module-checklist

If it passes these guidelines, and if you believe it’s a good PR otherwise, please add a comment with "shipit" in the text, and we will flag it for inclusion.

If it fails these guidelines, please add a comment with "needs_revision" in the text, along with the needed changes, and we will flag it for revision.

**Comment by gregdek** [comment]
@zeekin are you available to review this module?

**Comment by gregdek** [comment]
Looks like @zeekin is not available.

Cloud folks:

@jsmartin @scicoin-project @tombamford @garethr @jarv @jsdalton @silviud @adq @zbal @zeekin @willthames @lwade @carsongee @defionscode
@tastychutney @bpennypacker @loia

...is anyone available to take over maintainership of this module?

**Comment by gregdek** [comment]
Looks like not. Moving this module to core maintainership. Since this is now a core maintained module, evaluating this PR might take a while; @fvant thanks for your patience here.

File: cloud/amazon/ec2_metric_alarm.py [hunk 1]
```diff
@@ -184,8 +184,11 @@ def create_metric_alarm(connection, module):
         comparisons = {'<=' : 'LessThanOrEqualToThreshold', '<' : 'LessThanThreshold', '>=' : 'GreaterThanOrEqualToThreshold', '>' : 'GreaterThanThreshold'}
         alarm.comparison = comparisons[comparison]
 
-        dim1 = module.params.get('dimensions', {})
+        dim1 = module.params.get('dimensions')
```
Comment (willthames) [inline cloud/amazon/ec2_metric_alarm.py:187]: I don't understand this change at all. This code appears to be equivalent to the previous code, but less idiomatic.

Comment (willthames) [inline cloud/amazon/ec2_metric_alarm.py:187]: You say "Adding {} to module.params.get does not work". Is this a problem with a specific python version?

```
>>> a = dict(b='c', d='e')
>>> a.get('b', {})
'c'
>>> a.get('f', {})
{}
```

In what way does it not work?

**Comment by fvant** [comment]
I can not reproduce it (after 4 months worth of aws-sdk, boto and ansible upgrades).

Closing PR.

**Pull request closed without merging**